glide.lock

```diff
@@ -34,7 +34,7 @@ imports:
   - publicsuffix
 testImports:
 - name: github.com/davecgh/go-spew
-  version: 5215b55f46b2b919f50a1df0eaa5886afe4e3b3d
+  version: 2df174808ee097f90d259e432cc04442cf60be21
```

Comment:
@aayushKumarJarvis if this update is not required as part of this change, please revert this.